File: Samples/Android/VoiceAIKit/src/main/java/com/tencent/voiceai/kit/common/VoiceAIKitDefaults.kt
// Copyright (c) 2026 Tencent. All rights reserved.
// VoiceAIKit - 内置默认值（不对外开放配置）

package com.tencent.voiceai.kit.common

import com.tencent.voiceai.kit.VoiceAIKit

/**
 * VoiceAIKit 的内置默认值。
 *
 * 与 [VoiceAIKit.VoiceAIKitParams] 的分工：
 * - [VoiceAIKit.VoiceAIKitParams]：由接入方在启动时注入，是**外部输入**；
 * - 本类：Kit 内置的固定行为与默认取值，**不随参数开放配置**。
 */
internal object VoiceAIKitDefaults {

    /** 默认模型名（[VoiceAIKit.LLMParams] 的 model 未指定时使用）。 */
    const val DEFAULT_LLM_MODEL = "default"

    /** 对话系统提示词。 */
    const val SYSTEM_PROMPT =
        "你是一个友好的中文语音助手，回答要简洁、口语化，适合被朗读出来。"
}

File: Samples/iOS/VoiceAIKitDemo/VoiceAIKit/VoiceAIKitDefaults.swift
// Copyright (c) 2026 Tencent. All rights reserved.
// VoiceAIKit - 模块内部默认值 (对齐 Android VoiceAIKitDefaults.kt)

import Foundation

enum VoiceAIKitDefaults {
    /// 默认 LLM 模型名。
    static let defaultLlmModel = "default"

    /// 语音助手 system prompt：要求回答简洁、口语化，适合被朗读。
    static let systemPrompt = "你是一个友好的中文语音助手，回答要简洁、口语化，适合被朗读出来。"

    /// LLM 回复被打断且尚无内容时，气泡展示的占位提示。
    static let pausedHint = "已暂停生成"

    /// 会话历史最多保留条数（不含 system）。
    static let maxHistory = 20
}
